Source/WebCore/css/counterStyles.css

```diff
@@ -263,7 +265,7 @@
 
 @counter-style square {
     system: cyclic;
-    symbols: \25AA;
+    symbols: \25A0;
```

Comment:
Why is this different from the spec?

Reply:
Because WebKit has been using this symbol instead for square and I didn't want to change how it looks like.

Reply:
Can we move over the comment from RenderListMarker? 

`// CSS specification calls for U+25FE BLACK MEDIUM SMALL SQUARE; we have been using U+25A0 BLACK SQUARE instead for a long time.`